bin/cron4wma_crab_ds_access.sh

```diff
@@ -0,0 +1,122 @@
+##H
+##H Usage:
+##H    cron4wma_crab_ds_access.sh \
+##H        <keytab> value <amq> value <cmsmonitoring> value <stomp> value
+##H
+##H Example :
+##H    cron4wma_crab_ds_access.sh \
+##H        --keytab ./keytab --amq ./amq-creds.json --cmsmonitoring ./CMSMonitoring.zip --stomp ./stomp-v700.zip \
+##H        --p1 32000 --p2 32001 --host $MY_NODE_NAME --wdir $WDIR
+##H Arguments with values:
+##H   - keytab             : Kerberos auth file: secrets/kerberos
+##H   - amq                : AMQ credentials and configurations json file for the used AMQ topic
+##H   - cmsmonitoring      : dmwm/CMSMonitoring/src/python/CMSMonitoring folder as zip to be sent to Spark nodes
+##H   - stomp              : stomp.py==7.0.0 module as zip to be sent to Spark nodes which has lower versions.
+##H   - p1, p2, host, wdir : [ALL FOR K8S] p1 and p2 spark required ports(driver and blockManager), host is k8s node dns alias, wdir is working directory
+##H   - test                : will run only test job which will send only 10 documents to AMQ topic. Please give test/training AMQ credentials
+##H
+TZ=UTC
+START_TIME=$(date +%s)
+script_dir="$(
+    cd -- "$(dirname "$0")" >/dev/null 2>&1
+    pwd -P
+)"
+# get common util functions
+. "$script_dir"/utils/common_utils.sh
+
+trap 'onFailExit' ERR
+onFailExit() {
+    util4loge "finished with error!" || exit 1
+}
+# ------------------------------------------------------------------------------------------------------- GET USER ARGS
+unset -v KEYTAB_SECRET AMQ_JSON_CREDS CMSMONITORING_ZIP STOMP_ZIP PORT1 PORT2 K8SHOST WDIR IS_TEST help
+[ "$#" -ne 0 ] || usage
+
+# --options (short options) is mandatory, and v is a dummy param.
+PARSED_ARGS=$(getopt --unquoted --options v,h --name "$(basename -- "$0")" --longoptions keytab:,amq:,cmsmonitoring:,stomp:,p1:,p2:,host:,wdir:,test,help -- "$@")
+VALID_ARGS=$?
+if [ "$VALID_ARGS" != "0" ]; then
+    usage
+fi
+
+echo "$(date --rfc-3339=seconds)" "[INFO] Given arguments: $PARSED_ARGS"
+eval set -- "$PARSED_ARGS"
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+    --keytab)        KEYTAB_SECRET=$2     ; shift 2 ;;
+    --amq)           AMQ_JSON_CREDS=$2    ; shift 2 ;;
+    --cmsmonitoring) CMSMONITORING_ZIP=$2 ; shift 2 ;;
+    --stomp)         STOMP_ZIP=$2         ; shift 2 ;;
+    --p1)            PORT1=$2             ; shift 2 ;;
+    --p2)            PORT2=$2             ; shift 2 ;;
+    --host)          K8SHOST=$2           ; shift 2 ;;
+    --wdir)          WDIR=$2              ; shift 2 ;;
+    --test)          IS_TEST=1            ; shift   ;;
+    -h | --help)     help=1               ; shift   ;;
+    *) break ;;
+    esac
+done
+
+if [[ "$help" == 1 ]]; then
+    util_usage_help
+fi
+# ------------------------------------------------------------------------------------------------------------- PREPARE
+# Define logs path for Spark imports which produce lots of info logs
+LOG_DIR="$WDIR"/logs/$(date +%Y%m%d)
+mkdir -p "$LOG_DIR"
+
```

Comment:
The script should print all environment variable it uses, e.g. 
```
echo "LOG_DIR=$LOGDIR"
...
```
This will help you later in debug process.